Source/SteveSloth/Private/MyProjectileBaseClass.cpp

```diff
@@ -18,6 +18,23 @@ AMyProjectileBaseClass::AMyProjectileBaseClass()
 	ProjectileHitbox = CreateDefaultSubobject<UBoxComponent>(TEXT("Hitbox"));
 	AreaOfEffectHitbox = CreateDefaultSubobject<USphereComponent>(TEXT("AOEHitbox"));
 
+	Mesh = CreateDefaultSubobject<UStaticMeshComponent>(TEXT("Mesh"));
+
+	AreaOfEffectHitbox->SetSphereRadius(AreaOfEffectRadius);
```

Comment:
this one in begin play as well (since the area of effect radius hasnt been set yet).